renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
```

Comment:
should use the hmcts version: https://github.com/hmcts/terraform-module-template/blob/main/renovate.json